website/community/how-to-contribute/contribute-docs.md

```diff
@@ -21,5 +21,132 @@ sidebar_position: 1
 
 # Contribute Documentation
 
-TODO
+The [Fluss](https://alibaba.github.io/fluss-docs/) website presents Fluss and its community. It serves several purposes including:
 
+* Informing visitors about Fluss and its features as a streaming storage for real-time analytics
+* Encouraging visitors to download and use Fluss
+* Encouraging visitors to engage with the community
+* Providing comprehensive documentation for users and developers
+
+This guide provides information on how to contribute to the Fluss documentation.
+
+## Fork and Clone
+Fork the [Fluss repository](https://github.com/alibaba/fluss)
+    
+``` bash
+git clone https://github.com/<your-username>/fluss.git
+cd fluss
+```
+
+# Create a Branch
+    git checkout -b feature-branch-name
+
+# Install Dependencies
+   ```bash
+   cd website
+   npm install
+   ```
+
+# Directory Structure
+- The Fluss documentation is organized as follows:
+   ```
```

Comment:
Remove the indent. Currently, it is rendered in a light code color which is hard to read.